Description: Compatibility patch for Commons FileUpload
 Debian/Ubuntu ships a later version of commons-fileupload
 .
 This patch provides compatibility only.
Author: james.page@ubuntu.com
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/core/src/main/java/hudson/model/FileParameterValue.java
+++ b/core/src/main/java/hudson/model/FileParameterValue.java
@@ -40,6 +40,7 @@ import javax.servlet.ServletException;
 
 import org.apache.commons.fileupload.FileItem;
 import org.apache.commons.fileupload.disk.DiskFileItem;
+import org.apache.commons.fileupload.FileItemHeaders;
 import org.apache.commons.io.FilenameUtils;
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang.StringUtils;
@@ -277,5 +278,8 @@ public class FileParameterValue extends
         public OutputStream getOutputStream() throws IOException {
             return new FileOutputStream(file);
         }
+
+        public void setHeaders(FileItemHeaders headers) {
+        }
     }
 }
